src/RuleSet.php

```diff
@@ -21,7 +21,7 @@
  */
 final class RuleSet implements RuleSetInterface
 {
-    private $setDefinitions = array(
+    private static $setDefinitions = array(
```

Comment:
:-1: for staticness.
in general, i hate static things.

i agree that sometimes it's nice. but in all "nice" cases - we will basically use `const` instead of `static` with never PHP ;)

here - please revert. why? `setDefinitions` is not supposed to be constant, it is supposed to be extended by users in future (same like we register custom fixers, we want to register custom rules)